build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -266,5 +269,25 @@ private static void FixPermissions(string directory)
                 FS.FixModeFlags(directory);
             }
         }
+
+        private static string GetSharedFrameworkPublishPath()
+        {
+            string sharedFrameworkPublishPath = string.Empty;
+            
+            string preBuiltPortableBinaryPath=Environment.GetEnvironmentVariable("PORTABLE_BINARY_LOCATION")?.Trim();
```

Comment:
This should be read in Prepare.Init and set within the build context and then you should use BuildContext to read the value here.